<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogParamGlobal</class>
 <widget class="QDialog" name="DialogParamGlobal">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Paramétrage global</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/icones/icon</normaloff>:/icones/icon</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_1">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="GraphParamGlobal" name="graphParamGlobal" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Motif</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string notr="true">-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Minimum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Maximum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Modification</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Paramètre</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="comboMotif">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Manuel</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Linéaire ascendant</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Linéaire descendant</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Exponentiel ascendant</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Exponentiel descendant</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Aléatoire</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinRaideur">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>50.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinMin">
          <property name="minimum">
           <double>-20000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>20000.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinMax">
          <property name="minimum">
           <double>-20000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>20000.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboModif">
          <item>
           <property name="text">
            <string>Ajout</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Multiplication</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Remplacement</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboValeur"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Appliquer à l'étendue de vélocité</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinVelMin">
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinVelMax">
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushApplyToOthers">
         <property name="text">
          <string>Appliquer à d'autres instruments...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphParamGlobal</class>
   <extends>QWidget</extends>
   <header>dialog_paramglobal.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogParamGlobal</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogParamGlobal</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboMotif</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DialogParamGlobal</receiver>
   <slot>indexMotifChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinRaideur</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DialogParamGlobal</receiver>
   <slot>raideurChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushApplyToOthers</sender>
   <signal>clicked()</signal>
   <receiver>DialogParamGlobal</receiver>
   <slot>applyToOthers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinMin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DialogParamGlobal</receiver>
   <slot>minChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinMax</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DialogParamGlobal</receiver>
   <slot>maxChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>indexMotifChanged(int)</slot>
  <slot>raideurChanged(double)</slot>
  <slot>applyToOthers()</slot>
  <slot>minChanged(double)</slot>
  <slot>maxChanged(double)</slot>
 </slots>
</ui>
